src/averageAges.js

```diff
@@ -37,7 +45,21 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const targetPeople = people
+    .filter(person => person.sex === 'f')
```

Comment:
Consider using a more descriptive variable name instead of `targetPeople`. For example, `womenWithChildren`. 🤔